_posts/2024-06-06-switch-to-bam.md

```diff
@@ -13,10 +14,13 @@ contributors:
 reviewers:
   - pmenzel
   - hexylena
+last_modified_at: 2024-06-07
```

Comment:
shouldn't be necessary, we calculate this from git now :) we have the latest commit to identify last updated.

Reply:
I removed this front matter